### ISSUE TITLE
init: execv("/system/bin/init") failed: No such file or directory

### ISSUE THREAD

**Issue by Marietto2008** [antmicro/kvm-aosp-jetson-nano#15]
Hello to everyone.

I’m trying to virtualize Android on my jetson nano using qemu / kvm on ubuntu 18.04. This is the command that I give to start Android :

```
#!/bin/sh -e

qemu-system-aarch64 \
    -machine virt,gic-version=host -cpu host -smp 4 -enable-kvm \
    -append "console=ttyAMA0,38400 vt.global_cursor_default=0 androidboot.selinux=permissive debug drm.debug=0 androidboot.lcd_density=160" \
    -m 3048 \
    -kernel /root/Scrivania/Work/I9/aosp_kernel/out/android-4.14-q/common/arch/arm64/boot/Image \
    -initrd /root/Scrivania/Work/I9/aosp/out/target/product/generic_arm64/ramdisk.img \
    -drive index=0,if=virtio,format=raw,id=system,file=/root/Scrivania/Work/I9/aosp/out/target/product/generic_arm64/system.img \
    -drive index=1,if=virtio,format=raw,id=cache,file=/root/Scrivania/Work/I9/aosp/out/target/product/generic_arm64/cache.img \
    -drive index=2,if=virtio,format=raw,id=userdata,file=/root/Scrivania/Work/I9/aosp/out/target/product/generic_arm64/userdata.img \
    -netdev user,id=mynet,hostfwd=tcp::5401-:5555 -device virtio-net-pci,netdev=mynet \
    -device qemu-xhci \
    -device virtio-mouse-pci -device virtio-keyboard-pci \
    -d guest_errors \
    -serial mon:stdio \
    -device virtio-gpu-pci,virgl=on,xres=1280,yres=720 -display sdl,gl=on
```

it gives this error :

```
qemu-system-aarch64: warning: gic-version=host not relevant with kernel-irqchip=off as only userspace GICv2 is
supported. Using v2 ...

libGL error: MESA-LOADER: failed to open swrast (search paths /usr/lib/aarch64-linux-gnu/dri:\$${ORIGIN}/dri:/usr/lib/dri)
libGL error: failed to load driver: swrast
X Error of failed request:  GLXBadContext
  Major opcode of failed request:  149 (GLX)
  Minor opcode of failed request:  6 (X_GLXIsDirect)
  Serial number of failed request:  95
  Current serial number in output stream:  94
```

So I tried with :

`-device ramfb`

this is what happens : it reboots cyclically. each reboot it tells this :

https://pastebin.com/TF6D2D4B

maybe this error is caused by this (unfixed) error that I have explained here ? https://github.com/antmicro/kvm-aosp-jetson-nano/issues/14

**Comment by pgielda** [comment]
rsync is a tool for copying files (https://en.wikipedia.org/wiki/Rsync). If rsync fails it probably means that not all files were copied. You should not expect such an image to be working.
Also the error with libGL means android would not boot anyway... it requires virgl and virgl requires working libGL.

**Comment by Marietto2008** [comment]
one step at a time. First of all why you are not providing more help ? For example about how to fix the error #14. for the LibGL error i have already asked on the nvidia forum.

**Comment by pgielda** [comment]
Just trying to help, but I have no idea what you're doing wrong, just suggesting that your logs suggest that not all the files were copied. Trying to run such an image will most likely fail.

Also its just an open source tutorial, it does not come with 24/7 support.
